BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -510,8 +510,8 @@ - (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticatio
                         UIAlertView *challengeAlertView = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"Please Log In", @"Alert view title: title for SSO authentication challenge")
                                                                                      message:nil
                                                                                     delegate:self
-                                                                           cancelButtonTitle:NSLocalizedString(@"Cancel", @"Label: Cancel action. Usually used on buttons.")
-                                                                           otherButtonTitles:NSLocalizedString(@"Submit", @"Alert view button: submit button for SSO authentication challenge"), nil];
+                                                                           cancelButtonTitle:NSLocalizedString(@"Cancel", @"Label: cancel action, usually used on buttons")
```

Comment:
I think these are more readable when they are complete sentences. Why would we drop the capitalization and period at the end?